pkg/harness/tool/terraform/terraform.go

```diff
@@ -149,7 +149,7 @@ func (in *Terraform) Scan() ([]*console.StackPolicyViolationAttributes, error) {
 func (in *Terraform) plan() (string, error) {
 	output, err := exec.NewExecutable(
 		"terraform",
-		exec.WithArgs([]string{"show", in.planFileName}),
+		exec.WithArgs([]string{"show", in.planFileName, fmt.Sprintf("-parallelism=%d", *in.parallelism), fmt.Sprintf("-refresh=%t", *in.refresh)}),
```

Comment:
I don't see `parallelism` and `refresh` being assigned in the `Terraform` structure anywhere.

It should be checked for nil before appending and dereferencing.